test/src/main/java/org/axonframework/test/matchers/ExactClassMatcher.java

```diff
@@ -0,0 +1,38 @@
+package org.axonframework.test.matchers;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import static org.axonframework.common.BuilderUtils.assertNonNull;
+
+/**
+ * Matcher testing for exact class
+ *
+ * @author Yoann CAPLAIN
+ * @since 4.6.2
```

Comment:
For this change to be part of 4.6.2, the pull request should merge with the `axon-4.6.x` branch.
Hence, I think you need to rebase or recreate this pull request for that.